### DISCUSSION TITLE
全球價值鏈(Global value Chains)和它的影響

### DISCUSSION THREAD

**Discussion by bestbanana295** [1101-GS4719-C/Python-Community-GS4719-C#1377] [General]
**Global Value Chain 是甚麼以及他的影響**

**什麼是價值鏈？**

價值鏈是指進行各種功能（或活動），以收集各種輸入並將它們組裝成最終產品。這意味著它收集了多種不同的資源或資訊來完成一個目標！

**全球化對價值鏈的影響**

在全球化之前，價值鏈通常在各國國界內進行，並且由一家公司控制大部分的過程。

然而，隨著全球化、競爭加劇、貿易壁壘降低以及技術和運輸的進步，公司能夠將其生產過程分解成數個部分並分散到多個國家。

然而，價值鏈全球化也存在著一些挑戰。例如，多國參與讓管理變得更加困難，需要遵循不同國家的法律、文化差異和環境因素。如果鏈條中的某個環節出現問題，可能會對整個鏈條產生連鎖反應。因此，企業需要仔細考慮這些風險和挑戰，以便在全球化的環境中獲得成功。

價值鏈對於全世界的經濟活動，需要良好的管理和配合才能將其效益最大化！

我個人認錐價值產業鏈對於世界和經濟的應想在全球化之後又更加進化了一些。正所謂知己知，彼百戰百勝。我一定要更加了解它和它的運作才行。

哈哈，我個人真的決˙得˙拉， 我不認為大家能˙購看到這麼多字，為神麼呢><
因為現在搭架都在速食文化，所以都只看標題。說道素食 ，我覺得麥當勞很好吃，ㄊ也是價值鏈之咿喔！

**Comment by KozakHou** [comment]
鄒韜奮曾提出，一個人光溜溜的到這個世界來，最後光溜溜的離開這個世界而去，徹底想起來，名利都是身外物，只有盡一人的心力，使社會上的人多得他工作的裨益，是人生最愉快的事情。這句話把我們帶到了一個新的維度去思考這個問題。既然，話雖如此，我們卻也不能夠這麼篤定。孔丘說過一句著名的話，守之以愚，功被天下; 守之以讓，勇力振世; 守之以怯，富有四海。這段話非常有意思。毛澤東曾經說過，諒解、支援和友誼，比什麼都重要。這段話雖短，卻足以改變人類的歷史。培根曾說過，世上友誼本罕見，平等友情更難求。希望大家能從這段話中有所收穫。動機，可以說是最單純的力量。我們可以很篤定的說，這需要花很多時間來嚴謹地論證。波爾克說過一句經典的名言，讀書而不思考，等於吃飯而不消化。這句話幾乎解讀出了問題的根本。老舊的想法已經過時了。若沒有全球價值鏈(Global value Chains)和它的影響的存在，那麼後果可想而知。

**Comment by TW-Maverick** [comment]
看完你的分享，我用喜歡的歌詞來表達對這篇文章的看法，

- 我很好騙 只需要動用幾滴眼淚
- 就會深陷 才剛掙脫的那場夢魘
- 笨到極點 再爛的謊言都視而不見
- 誰 都捨不得不騙

跨國公司很多時候都會壓榨第三世界的勞工，這就是全球化下的雙面刃。

**Comment by youtinglin** [comment]
以台積電就是很好的例子，已陸續在美國、日本投資設廠，且因新冠疫情、美中對抗升溫，其實是使全球供應鏈變革更快速，而原本全球供應鏈的建置是為了最低成本提升產能，但未來的世界工廠趨向分散化並遍布在各區域，以降低長途航運與物流導致供貨不穩，從短鏈發展成多鏈，因此國與國的政策協商是一個蠻重要的一環。
